plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/expression/JdbcConnectorExpressionRewriterBuilder.java

```diff
@@ -55,8 +55,26 @@ public JdbcConnectorExpressionRewriterBuilder add(ConnectorExpressionRule<?, Str
         return this;
     }
 
+    public ExpressionMapping<JdbcConnectorExpressionRewriterBuilder> map(String pattern)
+    {
+        return new ExpressionMapping<>()
+        {
+            @Override
+            public JdbcConnectorExpressionRewriterBuilder to(String rewrite)
+            {
+                rules.add(new GenericRewrite(pattern, rewrite));
+                return JdbcConnectorExpressionRewriterBuilder.this;
+            }
+        };
+    }
+
     public ConnectorExpressionRewriter<String> build()
     {
         return new ConnectorExpressionRewriter<>(this.identifierQuote, rules.build());
     }
+
+    public interface ExpressionMapping<Continuation>
+    {
+        Continuation to(String rewrite);
```

Comment:
better name for the type parameter?

Reply:
i thought this is an awesome name. suggestions?

Reply:
Nothing else since there's a single usage today and I don't know what generic things this can accept. :) I'll mark resolved.

Reply:
Changed to `rewritePattern`  to match GenericRewrite's fields.